### PR TITLE
Book list

### PR DESCRIPTION
**Pull request by dsandoval11**
## Summary
- The book list view is created
- Roboto font is added

## Screenshot
image

## Trello Card
https://trello.com/c/fzsEMx9N/30-listado-de-libros

### CODE REVIEW

File: scss/application.scss [hunk 1]
```diff
@@ -1,3 +1,10 @@
 @import 'equalizer';
 @import 'styles';
 @import 'detail';
+@import 'list-book';
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap');
```
Comment (mcallegari10) [inline scss/application.scss:6]: This should be imported first 😄

File: scss/list-book.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
+@import 'colors';
```
Comment (mcallegari10) [inline scss/list-book.scss:1]: No need to import this in here. You can import it in the `application.scss` and all stylesheets can use it 😄

File: scss/list-book.scss [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    font-weight: bold;
+    margin-bottom: 8px;
+    margin-top: 12px;
+  }
+
+  .author {
+    color: $gray;
+    font-size: 14px;
+  }
+}
```
Comment (mcallegari10) [inline scss/list-book.scss:37]: Add an EOF line

File: scss/list-book.scss [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    max-height: 200px;
+    max-width: 142px;
+    width: 100%;
+  }
+
+  .title {
+    font-size: 16px;
+    font-weight: bold;
+    margin-bottom: 8px;
+    margin-top: 12px;
```
Comment (mcallegari10) [inline scss/list-book.scss:30]: Use margin shorthand: `margin: 12px 0 8px;`

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

File: scss/application.scss (hunk 1 above)
Comment (dsandoval11) [reply scss/application.scss:6]: Fixed in PR #6

File: scss/list-book.scss (hunk 3 above)
Comment (dsandoval11) [reply scss/list-book.scss:37]: Fixed in PR #6

File: scss/list-book.scss (hunk 4 above)
Comment (dsandoval11) [reply scss/list-book.scss:30]: Fixed in PR #6

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**